src/Symfony/Component/String/Resources/bin/common.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+define('LINE_WIDTH', 75);
+
+function bailout(string $message): void
+{
+    echo wordwrap($message, LINE_WIDTH)." Aborting.\n";
```

Comment:
error output should be sent to STDERR, not to STDOUT

Reply:
@stof In this case we should also do it in the Intl component?

Reply:
these are purely internal tools, let's keep `echo`.